app/models/invoice/data_presenter/line_item.rb

```diff
@@ -4,7 +4,7 @@ class Invoice
   class DataPresenter
     class LineItem < Invoice::DataPresenter::Base
       attributes :id, :added_tax, :currency, :included_tax, :price_with_adjustments, :quantity,
-                 :variant_id, :unit_price_price_and_unit, :unit_presentation,
+                 :variant_id, :unit_price, :unit_presentation,
```

Comment:
I think that this is a breaking change because old invoices would be stored with the old attribute and the new code doesn't know about it. But, I think that this feature is actually not used in production yet. Is that right, @RachL? Versioned invoices are not in production, right?

Reply:
I hope no one uses it, as it's not finished :sweat_smile: